presto-orc/src/test/java/com/facebook/presto/orc/DummyOrcDataSource.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.orc;
+
+import io.airlift.slice.FixedLengthSliceInput;
+
+import java.io.IOException;
+import java.util.Map;
+
+public class DummyOrcDataSource
```

Comment:
Let's keep the name `FakeOrcDataSource` since they both mean the same thing. Then the commit can be titled "Make FakeOrcDataSource a top level class"